src/System.Data.SqlClient/tests/FunctionalTests/System.Data.SqlClient.Tests.builds

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I did not notice that builds file because it was in the parent directory.  Is the expectation that for places where we split the test folder into a bunch of sub folders there's a .builds in the tests folder or do we want something SxS with the .csproj (I see we did it one way here but for some of the Http stuff there's no .builds files anywhere and they are also split).